src/Symfony/Bundle/WebProfilerBundle/Controller/ExceptionController.php

```diff
@@ -20,10 +20,14 @@
 use Twig\Error\LoaderError;
 use Twig\Loader\ExistsLoaderInterface;
 
+@trigger_error(sprintf('The "%s" class is deprecated since Symfony 4.4, use "%s" instead.', ExceptionController::class, ExceptionErrorController::class), E_USER_DEPRECATED);
+
 /**
  * ExceptionController.
  *
  * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @deprecated since Symfony 4.4, use the ExceptionErrorController instead.
```

Comment:
Now that the whole class is deprecated the method `templateExists` deprecation is useless

Reply:
Removed in https://github.com/symfony/symfony/pull/32741